x-pack/spec/config_management/elasticsearch_source_spec.rb

```diff
@@ -337,7 +337,7 @@
       end
 
 
-      %w(standard trial standard gold platinum).each do |license_type|
+      %w(trial standard gold platinum enterprise).each do |license_type|
```

Comment:
Why not use `LogStash::LicenseChecker::LICENSE_TYPES` ? to avoid dependency between config management and licence checker?

Reply:
because in this test we're checking that the all licenses except the "basic" license allow the use of this feature, and `LICENSE_TYPES` also contains "basic"

Reply:
it wasn't immediately clear to me that was something like "all licenses less than basic". Do you think `LICENSE_TYPES - ["basic"]` could be more meaningful?
After this final touch, LGTM